File: 0009-palindrome-number/0009-palindrome-number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        if(x<0) return false;
        long rx = 0;
        int temp = x;
        while(temp){
            rx = rx*10 + temp%10;
            temp /= 10; 
        }
        return x==rx;
    }
};

File: 0009-palindrome-number/0009-palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {
        if(x<0) return false;
        int rn = 0;
        int temp = x;
        while (temp !=0) {
            rn = rn * 10 + temp % 10;
            temp /= 10;
        }
        if (x == rn)
            return true;
        return false;
    }
}